docs/types/value-types.rst

```diff
@@ -111,7 +111,7 @@ the operands, division on integers always results in an integer.
 In Solidity, division rounds towards zero. This means that ``int256(-5) / int256(2) == int256(-2)``.
 
 Note that in contrast, division on :ref:`literals<rational_literals>` results in fractional values
-of arbitrary precision.
+of arbitrary precision. Here, the word "arbitrary" means "any" or "unlimited."
```

Comment:
As pointed out by @ekpyron this is really idiomatic and explaining the word here feels just weird to me. How about we just link to Wikpiedia for those who are not familiar with the term? I think this would be less obtrusive:

```suggestion
of `arbitrary precision <https://en.wikipedia.org/wiki/Arbitrary-precision_arithmetic>`_.
```
```
```
v